docs/usage-as-end-user.md

```diff
@@ -123,14 +123,14 @@ dnsPolicy: ClusterFirst
 dnsConfig:
   nameservers:
   - 8.8.8.8
-# Don't use pre-allocated data volumes. Defaults to 'false'.
-dontUsePreallocatedDataVolumes: true
+# Use dynamically allocated data volumes instead of pre-allocated data volumes. Defaults to 'false'.
+useDynamicallyAllocatedDataVolumes: true
```

Comment:
in this case shouldn't be default = true? You removed negation (word "don't"). What should be the default value? By default I think we want to have dynamically allocated data volumes.

Reply:
See above, `useDynamicallyAllocatedDataVolumes` is the same as `dontUsePreAllocatedDataVolumes`, i.e. the 2 options are: 
* pre-allocated data volumes (default)
* dynamically allocated data volumes

Reply:
So this statement is wrong 
`# Use dynamically allocated data volumes instead of pre-allocated data volumes. Defaults to 'false'.`

Reply:
my apologies, you're right I didn't read this correctly, see my comment https://github.com/gardener/gardener-extension-provider-kubevirt/issues/65#issuecomment-701445509